openslides/utils/autoupdate.py

```diff
@@ -187,6 +188,21 @@ def send_data(message):
                     {'text': json.dumps(output)})
 
 
+def send_collections_to_users(collections, users_ids):
```

Comment:
Maybe we can combine this code with the one used in send_data() so that we do not have duplicates?